wgsl/index.bs

```diff
@@ -992,6 +1006,13 @@ array_decoration
   </xmp>
 </div>
 
+<div class='example' heading='Access qualifier'>
+  <xmp>
+    var<storage> buf1 : [[access(read)]] Buffer;       # can read, cannot write.
+    var<storage> buf2 : [[access(read_write)]] Buffer; # can both read and write
+    var<uniform> b3 : Buffer;  # UBO. Hence: read-only. More restrictive layout rules. Does not need a qualifier.
```

Comment:
Sorry, I had a loosey-goosey example.

Better to be this:

```
    # Storage buffers
    var<storage> buf1 : [[access(read)]] Buffer;           # Can read, cannot write.
    var<storage> buf2 : [[access(read_write)]] Buffer; # Can both read and write

    #  Uniform buffer. Always read-only, and has more restrictive layout rules.
    var<uniform> params : ParamsTable; 
```

Reply:
done